django/cantusdb_project/main_app/views/chant.py

```diff
@@ -1632,15 +1628,25 @@ def get_chants_with_folios(chants_in_feast):
             current_chant = Chant.objects.filter(pk=pk).first()
             cantus_id = current_chant.cantus_id
 
-            request = requests.get(
-                f"https://cantusindex.org/json-cid/{cantus_id}",
-                timeout=5,
-            )
-            request_text = json.loads(request.text[2:])
-            if request_text:
-                context["suggested_fulltext"] = request_text[0]["fulltext"]
-            else:
-                context["suggested_fulltext"] = ""
+            try:
```

Comment:
Same as above....do we need to put something in this template that shows up in case suggested_fulltext = "" because of the failure?

Reply:
I think the template will look for "suggested_fulltext" in the context, not find it, and display nothing? I guess it could be better if we spell this out explicitly rather than handling it implicitly as is the case now.

Reply:
never mind, we already set `context["suggested_fulltext"]` five lines up. i don't think any changes are necessary here.

Reply:
I think if we don't have a suggested fulltext, we just don't suggest a fulltext (i.e. our suggestion should be `""`). It would get in people's ways if we added a different message.

Reply:
I guess the reason I ask is if the editor could just refresh the page and get a suggested text (because CantusIndex decided to work this time) is it worth telling them that?

Not a big deal, and I guess this keeps it agreeing with current Cantus